frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -241,9 +248,86 @@ function useCreateHmpbTargetElements({
 const BmdPaperBallotsMemoized = React.memo(BmdPaperBallots);
 const HandMarkedPaperBallotsMemoized = React.memo(HandMarkedPaperBallots);
 
+interface PrintingModalProps {
+  advancePrinting: () => void;
+  currentPrecinct: Precinct;
+  election: Election;
+  precinctIds: string[];
+  printIndex: PrintIndex;
+}
+
+function PrintingModal({
+  advancePrinting,
+  currentPrecinct,
+  election,
+  precinctIds,
+  printIndex,
+}: PrintingModalProps): JSX.Element {
+  if (printIndex.component === 'PaperChangeModal') {
+    return (
+      <Modal
+        centerContent
+        content={
+          <Prose textCenter>
+            <h1>Change Paper</h1>
+            <p>
+              Load printer with{' '}
+              <strong>
+                {getBallotLayoutPageSizeReadableString(election)}-size paper
+              </strong>
+              .
+            </p>
+          </Prose>
+        }
+        actions={
+          <Button onPress={advancePrinting} primary>
+            {getBallotLayoutPageSizeReadableString(election, {
+              capitalize: true,
+            })}{' '}
+            Paper Loaded, Continue Printing
+          </Button>
+        }
+      />
+    );
+  }
+  return (
+    <Modal
+      centerContent
+      content={
+        <Prose textCenter>
+          <p>
+            <Loading as="strong" wrapInProse={false}>
+              {`Printing L&A Package for ${currentPrecinct.name}`}
+            </Loading>
+          </p>
+          {precinctIds.length > 1 && (
+            <p>
+              This is package {printIndex.precinctIndex + 1} of{' '}
+              {precinctIds.length}.
+            </p>
+          )}
+          {getBallotLayoutPageSize(election) !== BallotPaperSize.Letter && (
+            <p>
+              Currently printing{' '}
+              {printIndex.component === 'HandMarkedPaperBallots'
+                ? getBallotLayoutPageSizeReadableString(election)
+                : 'letter'}
+              -size pages
```

Comment:
Looking forward to when we do L10n of voter components, we'll need to keep all phrases complete. I've found it helpful to just always default to full phrases for both L10n, readability, search/replace, etc. However since this is in VxAdmin — which won't be in multiple languages — you could ignore this suggestion! However I'd like to ask that we add punctuation to the end of this sentence. 😄 
```suggestion
              {printIndex.component === 'HandMarkedPaperBallots'
                ? `Currently printing ${getBallotLayoutPageSizeReadableString(election)}-size pages.`
                : 'Currently printing letter-size pages.'}
```

Reply:
Though we aren't planning to localize VxAdmin, I still like the idea of using complete phrases for readability! Added the punctuation, too ✅